packages/deployments/contracts/contracts/core/connext/facets/RoutersFacet.sol

```diff
@@ -249,86 +237,47 @@ contract RoutersFacet is BaseConnextFacet {
    * @param _router The relevant router address
    */
   function getRouterApprovalForPortal(address _router) public view returns (bool) {
-    return s.routerPermissionInfo.approvedForPortalRouters[_router];
+    return s.routerConfigs[_router].portalApproved;
   }
 
   // ============ Admin methods ==============
 
   /**
-   * @notice Used to set router initial properties
-   * @param router Router address to setup
-   * @param owner Initial Owner of router
-   * @param recipient Initial Recipient of router
+   * @notice Used to whitelist a given router
+   * @param _router Router address to setup
    */
-  function setupRouter(
-    address router,
-    address owner,
-    address recipient
-  ) external onlyOwnerOrRouter {
+  function addRouter(address _router) external onlyOwnerOrRouter {
```

Comment:
maybe we should change method names to `approveRouter` and `revokeRouter` but idk, open to either. Just important to disambiguate the two things as much as possible

Reply:
or `unapproveRouter` - we should follow the pattern for what we have for portals `approveRouterForPortals`, `unapproveRouterForPortals`...

Reply:
used same convention as portal